docs/src/examples/components/Icon/Usage/IconSetExample.shorthand.tsx

```diff
@@ -1,16 +1,27 @@
 import React from 'react'
 import { Grid, Icon, Provider } from '@stardust-ui/react'
 
+const cellStyles = {
+  margin: '10px 0',
+}
+
 const IconSetExampleShorthand = () => (
   <Provider.Consumer
     render={theme => (
-      <Grid columns={3} style={{ textAlign: 'center' }}>
+      <Grid columns={4} style={{ textAlign: 'center' }}>
```

Comment:
nit: just for the sake of introducing better separation of these groups, maybe the following markup could be considered (feel free to judge by yourself :) 

```jsx
const IconSetExampleShorthand = () => (
  <Provider.Consumer
    render={theme => (
      <>
        <div>
          <Divider>
            <Header as="h3" content="Regular"/>
          </Divider>
          <Grid columns={4} style={{ textAlign: 'center' }}>
            {Object.keys(theme.icons).map(name => (
              <div key={name} style={cellStyles}>
                <Icon name={name} />
                <br />
                <code>{name}</code>
              </div>
            ))}
          </Grid>
        </div>
        
        <div>
          <Divider>
            <Header as="h3" content="Outline"/>
          </Divider>
          <Grid columns={4} style={{ textAlign: 'center' }}>
            {Object.keys(theme.icons).map(name => (
              <div key={name + '-outline'} style={cellStyles}>
                <Icon name={name} variables={{ outline: true }} />
                <br />
                <code>{name} outline</code>
              </div>
            ))}
          </Grid>
        </div>
      </>
    )}
  />
)
```

![image](https://user-images.githubusercontent.com/9024564/48229736-04dfd400-e3ba-11e8-94af-09452f6837eb.png)

Reply:
Yes, I think this works better for now. Perhaps in the future would be good to show filled/outline versions side-by-side, but i suspect when enough icons go in we'll want to display in a different way anyways.